app/models/Harvest.scala

```diff
@@ -47,6 +47,17 @@ case class Harvest(id: Int,             // DB key
       .on('updated -> newDate, 'id -> id).executeUpdate()
     }
   }
+
+  def harvestUrl = {
```

Comment:
I'm going to move this logic back where it came from to keep the protocol specifics all in one place.